compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -3155,3 +3155,42 @@ class VolatileOnVal()(using Context)
 extends SyntaxMsg(VolatileOnValID):
   protected def msg(using Context): String = "values cannot be volatile"
   protected def explain(using Context): String = ""
+
+class UnusedSymbol(pos: SourcePosition,_msg: String, _actions: List[CodeAction])(using Context)
```

Comment:
Position is passed directly to `report.warning` as parameter so I think it should be removed from this place.

Reply:
:+1: removed